pims/base_frames.py

```diff
@@ -133,45 +133,145 @@ def __repr__(self):
                                   dtype=self.pixel_type)
 
 
-class FramesSequence(FramesStream):
-    """Baseclass for wrapping data buckets that have random access.
+class SliceableIterable(object):
```

Comment:
Oops. I changed api.py, actually. In the course of writing this class, I learned that iterable and iterator have [specific meanings](http://stackoverflow.com/a/9884259/1221924), and this is an iterable.